src/common/api.service.js

```diff
@@ -9,9 +9,23 @@ const ApiService = {
     Vue.use(VueAxios, axios)
     Vue.axios.defaults.baseURL = API_URL
   },
-  post(resource, params) {
+
+  setHeader (token) {
```

Comment:
There's probably a better solution for updating the header but this works now to get things going